spec/helpers/quadicon_helper_spec.rb

```diff
@@ -98,6 +98,7 @@
 # FIXME: complex describe blocks mirror the existing complex control flow
 
 describe QuadiconHelper do
+  include ApplicationHelper
```

Comment:
Should these also be `helper ApplicationHelper`?

Reply:
`helper` would be a first use for `ApplicationHelper`.. but it does work with either.

Should I change it?

Reply:
(can't tell the difference, because simply **mentioning** `ApplicationHelper` is apparently the important bit)